external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/OracleIntegrationSuite.scala

```diff
@@ -462,6 +464,9 @@ class OracleIntegrationSuite extends DockerJDBCIntegrationSuite with SharedSQLCo
       .option("lowerBound", "2018-07-04 03:30:00.0")
       .option("upperBound", "2018-07-27 14:11:05.0")
       .option("numPartitions", 2)
+      .option("oracle.jdbc.mapDateToTimestamp", "false")
```

Comment:
Do we need this line?

Reply:
Yes, we need this. Otherwise, Spark will read column `d` values as Catalyst type timestamp, which will fail the test.

<img width="1128" alt="screen shot 2018-09-21 at 3 18 00 pm" src="https://user-images.githubusercontent.com/12194089/45865915-9e730800-bdb1-11e8-9a42-a1394c601166.png">

Reply:
I see. We still have a date column in the input dataframe.